### ISSUE TITLE
[Adaptive Card iOS - Expense Report]: After activating '$300' button, Voice over focus is landing top of the screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#34]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over

**Repro Steps:**

Launch Adaptive Cards iOS application.
Home screen will appear.
Activate Voice over
Navigate to "V1.5" control and select it.
Navigate to 'Scenario' control and activate it.
Navigate to 'ExpenseReprot.JSON' and double tap to activate it.
The Card will open.
Navigate to the 'Air Travel Expense' button and double tap to activate it and observe the issue.

**Actual Result:**
After activating the 'Air Travel expense' button, voice over focus is landing on the top of the card and perfomring right swipe it is navigating from the starting.

**Observation:**
This issue is working fine with the keyboard navigation.

**Expected Result:**
On activating the 'Air Travel Expense' button, voice over focus should not land on the top of the card and either it should remain same on the button or it should be land on the Add your comment edit box.

**Note:**
Same issue is observed with Auto Mobile Expense and Excess Baggage Cost button.

**User Impact:**
This issue will impact the screen reader user as they have to put more effort to navigate on the next control if on activating the any control focus will land on the top of the card.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/1bf3c3e8-e42a-4cd7-b7bf-f85f1ac8e765
